HHS/Head-Start-TTADP
# [TTAHUB-1786] Training reports no events message

## Description of change

This adds a custom message for each tab of the training reports landing page letting the user know there are no events for their selected status.

Messages:

- You have no events with a “not started” status.
-  You have no events in progress.
- You have no completed events.
 

## How to test

View each tab of the events landing make sure the corresponding message is shown when applicable.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1786


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
